README.md

```diff
@@ -20,13 +20,50 @@ defp deps do
 end
 ```
 
+Modify `.formatter.exs` by adding patterns for all Surface files in `:inputs`:
+
+```elixir
+# .formatter.exs
+[
+  ...,
+  # match all .sface files and all .ex files with ~F sigils
+  inputs: ["lib/**/*.{ex,sface}", ...]
```

Comment:
Do we need to move this to v1.13 or greater section? I remember that the current formatter task read `surface_inputs` instead. Or I missing something?

Reply:
The current formatter uses `input` by default, but if you want the surface formatter to use separate inputs you can specify `surface_inputs`. That's why the README specifies it in the `## Formatting .sface files` section. (Now that you bring this up, we may need to update that. 🙂)

For the plugin, `surface_inputs` isn't an option. Does anyone know what happens if you specify `sface` files in `:inputs` under Elixir <= 1.12? In hindsight, I realize we possibly could have used `:inputs` all along. If that's true, it would probably make sense to remove `:surface_inputs` and only recommend `:inputs` in the docs.

Reply:
Unfortunately, I was able to confirm that matching `sface` files in `:inputs` in `.formatter.exs` causes `mix format` in Elixir 1.12 to break.

```
$ mix format
mix format failed for file: lib/foo.sface
** (SyntaxError) lib/foo.sface:1:1: syntax error before: '<'
```

So we should probably separate the instructions entirely for Elixir <= 1.12 and >= 1.13.

Reply:
I restructured the README so that the paths for installation and usage in Elixir 1.12 and 1.13 are detangled.

Reply:
To be clear — you were right all along @wingyplus :) Thanks again!￼